src/components/FAQPage.jsx

```diff
@@ -0,0 +1,171 @@
+// FAQPage.jsx
+import React, {Component} from 'react';
+import {Button} from "semantic-ui-react";
+
+class FAQPage extends Component {
```

Comment:
We need to direct users to this component from MainForm.jsx?

Reply:
@nmenego can we add a link to the header for this?